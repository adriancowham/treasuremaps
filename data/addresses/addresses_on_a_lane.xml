<posts>
<item rdf:about="http://sacramento.craigslist.org/gms/2339973909.html"> 
<title><![CDATA[Garage/Moving Sale  (208 Singer Lane Folsom CA 95630) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2339973909.html</link> 
<description><![CDATA[Moving out of US. Friday April 22 7:00 AM - 2:00PM and Saturday April 23 from 7:00 AM - 1:00PM.
<br> 

<br> 
kitchen items
<br> 
kids stuff
<br> 
Clothing
<br> 
out door items 
<br> 
Christmas items
<br> 
books
<br> 
Tools
<br> 
and more <!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=208 Singer Lane Folsom CA 95630 -->Location: 208 Singer Lane Folsom CA 95630
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date></dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339973909.html</dc:source> 
<dc:title><![CDATA[Garage/Moving Sale  (208 Singer Lane Folsom CA 95630) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued></dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339890102.html"> 
<title><![CDATA[Garage Sale - Friday Apr 22nd Only (East Roseville)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339890102.html</link> 
<description><![CDATA[FRI APRIL 22ND 8AM-NOON ONLY
<br> 

<br> 
3212 CORNICHE LANE, ROSEVILLE, CA 95661
<br> 

<br> 
Misc kids stuff, clothing, purses, small household items, stainless steel fridge...
<br> 

<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=East Roseville -->Location: East Roseville
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T20:12:34-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339890102.html</dc:source> 
<dc:title><![CDATA[Garage Sale - Friday Apr 22nd Only (East Roseville)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T20:12:34-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339661091.html"> 
<title><![CDATA[Large Yard Sale/ Garage Sale 4/23 (Roseville)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339661091.html</link> 
<description><![CDATA[Large 3 family yard sale- Saturday 4/23 beginning at 7am.  
<br> 

<br> 
810 Machado Lane, Roseville
<br> 
Riverside and Darling...at the end of Hernandez (Hernandez turns into Machado)
<br> 

<br> 
BBQ, TV, fans, lights, tons of kitchen stuff, clothes and more.  <!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Roseville -->Location: Roseville
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T17:25:40-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339661091.html</dc:source> 
<dc:title><![CDATA[Large Yard Sale/ Garage Sale 4/23 (Roseville)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T17:25:40-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339587790.html"> 
<title><![CDATA[Estate farm sale (Auburn) $1]]></title> 
<link>http://sacramento.craigslist.org/gms/2339587790.html</link> 
<description><![CDATA[Entire farm sale, inside and out. Furniture, antiques, old cars, tractors, farm tools. Too much to list. Saturday  April 23, 6:00am til gone. 701 Doe Run ln.<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Auburn -->Location: Auburn
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T16:42:01-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339587790.html</dc:source> 
<dc:title><![CDATA[Estate farm sale (Auburn) $1]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T16:42:01-07:00</dcterms:issued> 
</item> 
<item rdf:about="http://sacramento.craigslist.org/gms/2339507822.html"> 
<title><![CDATA[ANNUAL PLANT & GARAGE SALE (Granite Bay)]]></title> 
<link>http://sacramento.craigslist.org/gms/2339507822.html</link> 
<description><![CDATA[DON'T MISS OUT ON THIS ANNUAL PLANT & GARAGE SALE
<br> 

<br> 
FRIDAY April 22nd     From 7:00am to 4:00pm
<br> 
AND
<br> 
SATURDAY April 23rd    From 7:00am to 12 Noon
<br> 

<br> 
IT ONLY HAPPENS ONCE A YEAR. BEAUTIFUL ROSES.  Many different kinds of plants
<br> 
6485 Countrywoods Lane
<br> 
Granite Bay<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=Granite Bay -->Location: Granite Bay
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-21T15:54:19-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2339507822.html</dc:source> 
<dc:title><![CDATA[ANNUAL PLANT & GARAGE SALE (Granite Bay)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-21T15:54:19-07:00</dcterms:issued> 
</item>
<item rdf:about="http://sacramento.craigslist.org/gms/2350440927.html"> 
<title><![CDATA[Huge yard sale (lincoln)]]></title> 
<link>http://sacramento.craigslist.org/gms/2350440927.html</link> 
<description><![CDATA[201 oak tree lane
<br> 
lincoln ca. 95648 
<br> 
McBean to 193
<br> 
first right east of ferrari ranch rd. 
<br> 
first driveway to the left
<br> 

<br> 
7;30am-4pm friday, saturday and sunday no early birds!!!!!!!
<br> 

<br> 

<br> 
so much I cant list nearly all. something for everyone
<br> 
prices for everyone too!!!!!!!!!!!!!
<br> 

<br> 
refridgerator, 2 washers and 2 dryers
<br> 
furniture
<br> 
riding lawn mower and push mower
<br> 
tools 
<br> 
sports equip
<br> 
books, games, nicknaks
<br> 
golf clubs
<br> 
computers
<br> 
vidio games
<br> 
lots of electronics
<br> 
lots and lots more
<br> 

<br> 

<br> 
<!-- START CLTAGS --> 
 
 
<br><br><ul class="blurbs"> 
<li> <!-- CLTAG GeographicArea=lincoln -->Location: lincoln
<li>it's NOT ok to contact this poster with services or other commercial interests</ul> 
<!-- END CLTAGS -->]]></description> 
<dc:date>2011-04-27T21:18:23-07:00</dc:date> 
<dc:language>en-us</dc:language> 
<dc:rights>Copyright &#x26;copy; 2011 craigslist, inc.</dc:rights> 
<dc:source>http://sacramento.craigslist.org/gms/2350440927.html</dc:source> 
<dc:title><![CDATA[Huge yard sale (lincoln)]]></dc:title> 
<dc:type>text</dc:type> 
<dcterms:issued>2011-04-27T21:18:23-07:00</dcterms:issued> 
</item>
</posts>