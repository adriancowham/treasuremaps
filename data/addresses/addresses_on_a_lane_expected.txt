208 Singer Lane
3212 CORNICHE LANE
810 Machado Lane
701 Doe Run ln
6485 Countrywoods Lane
201 oak tree lane